components/_patterns/02-molecules/menus/_menu.twig

```diff
@@ -43,7 +43,9 @@
     {% embed "@atoms/03-lists/00-ul.twig" with {
       ul_base_class: menu_class,
       ul_modifiers: menu_modifiers,
-      ul_blockname: menu_blockname
+      ul_blockname: menu_blockname,
+      ul_role: menu_role,
+      ul_rel: menu_rel
```

Comment:
Not a huge deal, but the last line should end in a comma to make the diffs cleaner. In this case, line 46 looks like it's been deleted, and a new line added, but it actually just the addition of the comma.